src/StartMenu.ts

```diff
@@ -296,64 +298,85 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const headerText = this.desc.name === 'Genes'? 'Gene Sets' : `${this.desc.name.substr(0, this.desc.name.length - 1)} Panels`;
```

Comment:
you could add this meta data `Gene Sets` vs `${this.desc.name.substr(0, this.desc.name.length - 1)} Panels` plugin meta data, defined in `phovea.js`